frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -47,6 +47,51 @@ const queries = (() => Object.values(CAPACITY_INFO_QUERIES))();
 const parser = compose((val) => val?.[0]?.y, getInstantVectorStats);
 const getProvisionedCapacity = (value: number) => (value % 1 ? (value * 3).toFixed(2) : value * 3);
 
+const CurrentlyUsedCapacity: React.FC<{ currentCapacity: any; t: any }> = ({
+  currentCapacity,
+  t,
+}) => (
+  <TextContent className="pf-u-font-weight-bold pf-u-secondary-color-100 ceph-add-capacity__current-capacity">
+    {t('ceph-storage-plugin~Currently Used:')}&nbsp;
+    {currentCapacity}
+  </TextContent>
+);
+
+const RawCapacity: React.FC<{ t: any; osdSizeWithoutUnit: any; replica: any }> = ({
+  t,
+  osdSizeWithoutUnit,
+  replica,
+}) => {
+  const provisionedCapacity = getProvisionedCapacity(osdSizeWithoutUnit);
+  return (
+    <>
+      <FormGroup
+        fieldId="request-size"
+        label={t('ceph-storage-plugin~Raw Capacity')}
+        labelIcon={<FieldLevelHelp>{requestedCapacityTooltip(t)}</FieldLevelHelp>}
+      >
+        <TextInput
+          isDisabled
+          id="request-size"
+          className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
+          type="number"
+          name="requestSize"
+          value={osdSizeWithoutUnit}
+          aria-label="requestSize"
+          data-test-id="requestSize"
+        />
+        <TextContent className="ceph-add-capacity__provisioned-capacity">
+          {' '}
+          {t('ceph-storage-plugin~x {{ replica, number }} replicas =', {
+            replica,
+          })}{' '}
+          <strong data-test="provisioned-capacity">{provisionedCapacity}&nbsp;TiB</strong>
+        </TextContent>
+      </FormGroup>
+    </>
+  );
+};
+
```

Comment:
components should not be inside components.
move them outside.

Reply:
I have written it outside only

Reply:
okay, my bad!